processor/elasticinframetricsprocessor/go.mod

```diff
@@ -3,43 +3,54 @@ module github.com/elastic/opentelemetry-collector-components/processor/elasticin
 go 1.21.1
 
 require (
-	github.com/elastic/opentelemetry-lib v0.0.0-20240604140721-08863a456d6c
+	github.com/elastic/opentelemetry-lib v0.0.0-20240607090112-76a6d08387f0
```

Comment:
There is a latest release version for this library now. Maybe we can use that .

Reply:
Sounds good, bumped to v0.1.0 in https://github.com/elastic/opentelemetry-collector-components/pull/10/commits/ae72d652c70305b5886ffe30aa9878d2e7aefbb5